repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -156,7 +158,7 @@ class Libmesh(AutotoolsPackage):
     depends_on("slepc", when="+slepc")
     depends_on("petsc", when="+petsc")
     depends_on("tbb", when="threads=tbb")
-    depends_on("vtk", when="+vtk")
+    depends_on("vtk+mpi", when="+vtk")
```

Comment:
@capitalaslash, you should add a justification why this change is needed to the PR or the code.

Reply:
Ah, found it there.
> libmesh would silently disable some optional libraries when they are set via a simple --libname-enable switch. Some dependencies offer a *-required variant that enforces their activation.
From libmesh configure script, as an example for petsc:
--enable-petsc-required Error if PETSc is not detected by configure
>  This uncovered an error in the vtk dependency that was not truly ever enabled until the --vtk-required was added, leading to the change to the dependency to vtk+mpi